frontend/benefit/applicant/src/components/applications/forms/application/step2/ApplicationFormStep2.tsx

```diff
@@ -478,7 +478,6 @@ const ApplicationFormStep2: React.FC<DynamicFormStepComponentProps> = ({
 
             <$GridCell $colSpan={12}>
               <Heading
-                as="h3"
```

Comment:
Why did you have to remove it?

Reply:
Probably an accident. I try to put it back.

Reply:
reverted

Reply:
Hmm. After merge it started to nag about it:
```
@frontend/applicant: Type error: Type '{ as: string; size: "xs"; header: string | undefined; tooltip: string | undefined; }' is not assignable to type 'IntrinsicAttributes & HeadingProps & { children?: ReactNode; }'.
@frontend/applicant:   Property 'as' does not exist on type 'IntrinsicAttributes & HeadingProps & { children?: ReactNode; }'.
@frontend/applicant:   536 |             <$GridCell $colSpan={12}>
@frontend/applicant:   537 |               <Heading
@frontend/applicant: > 538 |                 as="h3"
@frontend/applicant:       |                 ^
@frontend/applicant:   539 |                 size="xs"
@frontend/applicant:   540 |                 header={t(`${translationsBase}.heading5EmploymentSub1`)}
@frontend/applicant:   541 |                 tooltip={t(

```
I will check what to do with it.

Reply:
The Heading component's `as` property eas simply dropped out, so I added it back. Now it works.